### PR TITLE
Only auto-detect bonds if we have 3D coordinates

### PR DESCRIPTION
**Pull request by psavery**
Signed-off-by: Patrick Avery <patrick.avery@kitware.com>

### CODE REVIEW

**Comment by psavery** [comment]
@cjh1 This one also.

File: girder/molecules/server/openbabel.py [hunk 1]
```diff
@@ -55,6 +55,12 @@ def cjson_to_ob_molecule(cjson):
     return mol
 
 def autodetect_bonds(cjson):
+    # Only autodetect bonds if we have 3D coordinates
+    if ('atoms' not in cjson or 'coords' not in cjson['atoms'] or
+        '3d' not in cjson['atoms']['coords'] or
+        len(cjson['atoms']['coords']['3d']) == 0):
```
Comment (cjh1) [inline girder/molecules/server/openbabel.py]: Lets use jsonpath to make the check a little more readable, something like:

`parse('atoms.coords.3d').find(cjson)`

The are lots of examples, in the repo ...

Comment (psavery) [reply girder/molecules/server/openbabel.py]: Okay.

**Comment by psavery** [comment]
I've tried this out in the notebook, and it seems to work. Thanks @cjh1.

**Pull request merged**